redis/pubsub_example_test.go

```diff
@@ -116,9 +118,18 @@ func publish() {
 	}
 	defer c.Close()
 
-	c.Do("PUBLISH", "c1", "hello")
-	c.Do("PUBLISH", "c2", "world")
-	c.Do("PUBLISH", "c1", "goodbye")
+	if _, err = c.Do("PUBLISH", "c1", "hello"); err != nil {
+		fmt.Println(err)
```

Comment:
is there a reason for using `fmt.Println` here over say `t.Log()`?

Reply:
It's because this is an example not a test, so trying to be more real.